### DISCUSSION TITLE
Enable customization for Navigation bar background with support for any color

### DISCUSSION THREAD

**Discussion by vega99** [jpudysz/react-native-unistyles#199] [Feature Requests]
Hello!

Firstly, I want to express my gratitude for this remarkable library. I've never encountered anything quite like it in React Native; it's truly a game-changer for styling. This library has not only enhanced the aesthetics but also significantly improved code readability for me.

Now, I'm curious if there's a way to set any color as the background for the Android navigation bar. Perhaps we could introduce a parameter to determine if the background color should be light or dark? Looking forward to your insights on this!

**Comment by jpudysz** [comment]
Hey and thank you 👋
It's already implemented, you can check the docs here: 

https://reactnativeunistyles.vercel.app/reference/theming/#update-navigationbar-color-during-runtime
